src/App/App.tsx

```diff
@@ -1,42 +1,119 @@
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 import USAState from '../USAState/USAState';
 import ComparisonContainer from '../ComparisonContainer/ComparisonContainer';
 import Stat from '../Stat/Stat';
+import { fetchAllCurrentUSAData, fetchCurrentStateData } from '../apiCalls'
+import { GiCoffin } from 'react-icons/gi';
+import { RiVirusFill, RiHospitalFill } from 'react-icons/ri';
 import './App.scss';
 
 interface Props { 
 }
 
 interface State {
-  selectedUSAState: {};
+  allUSAData: Details, 
+  selectedUSAState: Details
 }
 
-class App extends Component<Props,State> {
+interface Details {
+  date?: number,
+  state?: string,
+  positive: number,
+  hospitalizedCurrently?: number,
+  death: number
+}
+
+
+class App extends Component<Props, State> {
   constructor(props: Props) {
     super(props);
     this.state = {
-      selectedUSAState: {}
+      allUSAData: {
+        positive: 0,
+        death: 0
+      }, 
+      selectedUSAState: {
```

Comment:
We might want to reach out to a mentor to confirm if this is the best initial state data.